api/src/main/java/graphql/nadel/instrumentation/parameters/NadelInstrumentRootExecutionResultParameters.java

```diff
@@ -13,12 +14,19 @@
 
     private final ExecutionContext executionContext;
     private final InstrumentationState instrumentationState;
-    private final NormalizedQueryFromAst normalizedQueryFromAst;
+    @Nullable
+    private final ExecutableNormalizedOperation query;
```

Comment:
adding `ExecutableNormalizedOperation` so the tracer lib can resolve the types from the execution result (which is a Json map)

Reply:
still need to add `__typename` to interfaces and unions

Reply:
> still need to add __typename to interfaces and unions

ok, that's done